shared/common-adapters/popup-menu.native.js

```diff
@@ -19,8 +19,8 @@ const MenuRow = (props: MenuItemProps) => (
   <TouchableOpacity
     disabled={!props.onClick}
     onPress={() => {
+      props.onHidden() // auto hide when making a selection
```

Comment:
Re-ordered this so whatever potential side-effects `onHidden` has doesn't happen after `onClick` (in this case, navigating up right after navigating to the add people page)